Kernel/Config/Files/XML/Ticket.xml

```diff
@@ -128,7 +128,7 @@
         </Value>
     </Setting>
     <Setting Name="Ticket::Service::KeepChildren" Required="1" Valid="1">
-        <Description Translatable="1">Retains all services in listings even if they are children of invalid elements.</Description>
+        <Description Translatable="1">Enable if you want valid child services of invalid services to be treated as valid. When disabled, any descendant service of invalid service will be treated as invalid regardless of this descendant own validity.</Description>
```

Comment:
```suggestion
        <Description Translatable="1">Retains all services in listings even if they are children of invalid elements.</Description>
```
Unfortunately, your description is more difficult to understand. 
Therefore, I would prefer the original one.

Reply:
done